runtest.py

```diff
@@ -112,7 +113,14 @@ def __init__(self, **kwargs):
         else:
             self._firefox_profile_path = ""
 
-        # chrome config
+        # init status job id path, if not specify in exec config, default will be empty string
+        status_job_id = self.exec_config.get("status-job-id", "")
+        if status_job_id:
+            self.status_job_id_path = os.path.join(StatusFileCreator.get_status_folder(), self.exec_config.get("status-job-id", ""))
+        else:
```

Comment:
We will get either "" or content from `status_job_id = self.exec_config.get("status-job-id", "")`.
The if-else here can be removed?